postmark/django_backend.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, *args, **kwargs):
         else:
             self.track_opens = getattr(settings, 'POSTMARK_TRACK_OPENS', False)
 
+        if 'message_stream' in kwargs:
+            self.message_stream = kwargs['message_stream']
```

Comment:
Did you also test the case where you can pass `message_stream` into the EmailMessage kwargs? As opposed to setting it on the instance after instantiation? I ask because these other kwargs `track_opens` and `tag` del the kwarg after

```
del kwargs['tag']
```

Thinking we probably need to do the same here as it might raise an error upstream.